src/test/resources/application-local.yaml

```diff
@@ -0,0 +1,6 @@
+targetInstance: https://hmi-apim-svc-test.azure-api.net
+targetHost:  hmi-apim-svc-test.azure-api.net
+targetSubscriptionKey: samplesecret
+hearingApiRootContext: /hmi/hearings
+hmiApiRootContext : /hmi/
+spring.flyway.enabled: false
```

Comment:
do you need this, we don't hold any schemas to migrate right?

Reply:
Yes. It makes local development easy. It's a standard practice with spring boot.

Reply:
Removed.